cmd/photobak/main.go

```diff
@@ -33,6 +34,7 @@ var (
 func init() {
 	flag.StringVar(&repoDir, "repo", repoDir, "The directory in which to store the downloaded media")
 	flag.BoolVar(&keepEverything, "everything", keepEverything, "Whether to store all metadata returned by API for each item")
+	flag.BoolVar(&skipRepoChecks, "skiprepochecks", skipRepoChecks, "Don't check items that already exist in the database")
```

Comment:
I think, that it definitely should be off by default, because in most of use cases this is exactly the user wants. OK, I'll rewrite it to make it off by default and be enabled by `integrity-check` option.

JFYI: I used `skiprepochecks` instead of `skip-repo-checks` to follow already established style (`googlephotos`, `maxalbums`, `maxphotos`).